CacheManager.StackExchange.Redis.SelfCleaning/SelfCleaningRedisConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using CacheManager.Core;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning
+{
+    public static class SelfCleaningRedisConfigurationBuilderExtensions
```

Comment:
(After doing this https://github.com/reverser-gever/CacheManager.StackExchange.Redis.SelfCleaning/pull/2#discussion_r394583105) Add (short) summary about the proper way of using this class - "First call 'WithSelfCleaningRedisConfiguration' and then 'WithSelfCleaningRedisCacheHandle' "

Reply:
Added